packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -92,10 +92,11 @@ const DevApp: FunctionComponent = () => {
       </div>
 
       <MockBlockDock
-        debug
         blockDefinition={blockDefinition}
         blockEntity={blockEntity}
         blockName="Test Block"
+        debug
+        key={testBlockType} // completely reset the state when block type has changed
```

Comment:
there are a few bugs when running MBD in dev mode (i.e. its own dev mode, which doesn't end up in templates) and switching between block types